corehq/apps/app_manager/models.py

```diff
@@ -3916,6 +3916,7 @@ class ApplicationBase(LazyBlobDoc, SnapshotMixin,
     built_on = DateTimeProperty(required=False)
     build_comment = StringProperty()
     comment_from = StringProperty()
+    last_released = DateTimeProperty(required=False)
```

Comment:
PR title says `released_on` but you added `last_released`.
I think `released_on` is a better name since this is expected to be on Builds.

Reply:
There's a conversation about this on the spec. Mobile team asked for a different name.

Reply:
Re-reading the mobile team's question, they only need the header to be a different name, so I'm going to leave this alone. The attribute is `last_released` because `released_on` doesn't make it clear whether it's the first time released, last time released, or something else.

Reply:
`last_released` would make sense if we were talking about the application and `released_on` makes sense when we are talking about a build. This value is set on builds, so i would still vote for `released_on`.